addons/sale_timesheet/report/timesheets_analysis_report.py

```diff
@@ -0,0 +1,45 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models, api
+
+class TimesheetsAnalysisReport(models.Model):
+
+    _inherit = "timesheets.analysis.report"
+    _auto = False
+
+    so_line = fields.Many2one("sale.order.line", string="SO line", readonly=True)
+    timesheet_invoice_type = fields.Selection([
+        ("billable_time", "Billed on Timesheets"),
+        ("billable_fixed", "Billed at a Fixed price"),
+        ("non_billable", "Non Billable Tasks"),
```

Comment:
I added two more tuples in this field, can you update? 🙂 
Or we could create a variable containing this array in sale_timesheet/models/account.py and use it in this report (see [here](https://github.com/odoo/odoo/blob/42ce97a4797c8c1ee0f42fb44ed51d7e621b38ab/addons/sale_project/models/product.py#L8-L13) an example.). 🙂